### PR TITLE
💅

### PR DESCRIPTION
**Pull request by dawitelias**
More UI updates!

| Dark | Light |
| ------------- |-------------|
| 374F6B20-AA00-4437-9353-9D3756890B73 | IMG_A20B7FF1316E-1 |

### CODE REVIEW

File: Workouts/HomeView.swift [hunk 1]
```diff
@@ -25,15 +25,19 @@ struct HomeView: View {
...
+                    }) {
+                        ZStack {
                             FeaturedWorkout(workout: featuredWorkout!)
-                                .padding(EdgeInsets(top: 0, leading: 10, bottom: 0, trailing: 10))
+                            NavigationLink(destination: WorkoutDetail(workout: featuredWorkout!)) {
+                                EmptyView()
+                            }
                         }
-                    }.padding(.trailing, -22.0) // <-- hide that dang little arrow icon
+                    }
```
Comment (dawitelias) [inline Workouts/HomeView.swift:40]: @emilyCheroske Found a hack online for hiding the disclosure indicator that seems a little better: https://nukedbit.dev/hide-disclosure-arrow-indicator-on-swiftui-list/.

And now it doesn't interfere with the map view.

File: Workouts/HomeView.swift [hunk 2]
```diff
@@ -52,6 +56,7 @@ struct HomeView: View {
                     }
                 }
             }
+            .listStyle(GroupedListStyle()).environment(\.horizontalSizeClass, .regular)
```
Comment (dawitelias) [inline Workouts/HomeView.swift:59]: @emilyCheroske Stole that new list/tableview style from Jeff and Justin's project. I think it looks pretty good!

File: Workouts/SupportingViews/FeaturedWorkout.swift [hunk 3]
```diff
@@ -25,13 +25,12 @@ struct FeaturedWorkout: View {
...
                 }
             }
+            .padding(.top)
             MapView(workout: workout)
                 .frame(height: 200)
-                .cornerRadius(12.0)
-            Divider()
-                .padding(.top)
+                .padding(.horizontal, -15)
+                .padding(.bottom, -6)
```
Comment (dawitelias) [inline Workouts/SupportingViews/FeaturedWorkout.swift:32]: @emilyCheroske Had to do some weird negative padding stuff to account for the space the disclosure indicator is no longer taking up.

File: Workouts/HomeView.swift (hunk 2 above)
Comment (emilyCheroske) [reply Workouts/HomeView.swift:59]: Dude it looks awesome!

**Review by emilyCheroske — APPROVED** [review]
🎉 This looks so sweet dude!!!! Nice

**Comment by emilyCheroske** [comment]
I wonder how hard it would be to scrape together images that would replace the map view for workout types that don't have location data -- or, just what we could put there to replace the maps for those types

**Comment by dawitelias** [comment]
> I wonder how hard it would be to scrape together images that would replace the map view for workout types that don't have location data -- or, just what we could put there to replace the maps for those types

Good point. Relevant images or gifs (if want to keep it silly) could work!

**Pull request merged**